alternator/executor.cc

```diff
@@ -959,7 +998,7 @@ static future<executor::request_return_type> create_table_on_shard0(tracing::tra
     }
     builder.with_column(bytes(executor::ATTRS_COLUMN_NAME), attrs_type(), column_kind::regular_column);
 
-    verify_billing_mode(request);
+    billing_mode_type bm = verify_billing_mode(request);
```

Comment:
nitpick: if you call this variable billing_mode instead of bm, it's use below will be more self-explanatory.